### PR TITLE
More details about fees and core pools redirection

### PR DESCRIPTION
**Pull request by Tritium-VLK**

### CODE REVIEW

File: docs/concepts/governance/multisig.md
Comment (Tritium-VLK) [inline docs/concepts/governance/multisig.md]: @markusbkoch Your name is being connected to this signer address on the LM multisig here.  Just want to make sure to get sign-off from you before pushing that to the docs pages.

File: docs/concepts/governance/bal-token.md
Comment (mikemcdonald) [inline docs/concepts/governance/bal-token.md]: Any reason why this link is removed?

File: docs/concepts/governance/bal-token.md
Comment (mikemcdonald) [inline docs/concepts/governance/bal-token.md]: This should probably keep a header format

File: docs/concepts/governance/bal-token.md
Comment (mikemcdonald) [inline docs/concepts/governance/bal-token.md]: This link is broken. Isn't the original correct?

File: docs/concepts/governance/multisig.md
Comment (mikemcdonald) [inline docs/concepts/governance/multisig.md]: The original `Multisig` seems fine to me

**Review by mikemcdonald — CHANGES_REQUESTED** [review]

File: docs/concepts/governance/bal-token.md
Comment (Tritium-VLK) [reply docs/concepts/governance/bal-token.md]: Correct.  Pushed changes.

File: docs/concepts/governance/bal-token.md
Comment (Tritium-VLK) [reply docs/concepts/governance/bal-token.md]: Yeah it is.  Was merging in some changes from others here.

**Comment by Tritium-VLK** [comment]
@mikemcdonald thanks for catching all that.  Won't merge a bunch of other peoples ideas together like that again. Showing @0xdanko how to use github.

**Comment by mikemcdonald** [comment]
Address for polygon ecosystem fund is still pointing to mainnet. Once that's fixed I'll get this merged

**Review by mikemcdonald — APPROVED** [review]

**Pull request merged**
